src/ci/docker/dist-various-2/Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN /tmp/build-solaris-toolchain.sh sparcv9 sparcv9 solaris-sparc
 COPY dist-various-2/build-x86_64-fortanix-unknown-sgx-toolchain.sh /tmp/
 # We pass the commit id of the port of LLVM's libunwind to the build script.
 # Any update to the commit id here, should cause the container image to be re-built from this point on.
-RUN /tmp/build-x86_64-fortanix-unknown-sgx-toolchain.sh "53b586346f2c7870e20b170decdc30729d97c42b"
+RUN /tmp/build-x86_64-fortanix-unknown-sgx-toolchain.sh "63b1b1c5b1ca008099095aa0f8a081a80c8d27f4"
```

Comment:
5125c169b30837208a842f85f7ae44a83533bd0e

Reply:
This whole thing strikes me as quite fragile ...